documentation/Cohen Hubbard Oesterwinter.tex

```diff
@@ -0,0 +1,166 @@
+\[
+f^{(m)}\of{x_0} ≅ \sum{\gn = 0}[n]{\gd^{m}_{n,\gn}\of{\VectorSymbol{\ga}} f\of{\ga_\gn}}
+\]
+where $m ≤ n$ and the coefficients $\gd^{m}_{n,\gn}$ are dependent on $\VectorSymbol{\ga}$ but independent of $f$.
+
+If the $\VectorSymbol{\ga}$ are equally spaced with step $-h$ (\idest, $\ga_\gn = x_0 - \gn h$) then we can write a backward formula as follows:
+\begin{equation}
+f^{(m)}\of{x_0} = \sum{\gn = 0}[n]{\gd^{m}_{n,\gn}\of{\VectorSymbol{\ga}} f\of{x_0 - \gn h}} + \BigO\of{h^{n - m + 1}}
+\label{fornbergbackwardpreliminary}
+\end{equation}
+
+In this case equation (3.8) from \cite{Fornberg1988} may be rewritten as follows, restoring $x_0$:
+\begin{align}
+\gd^{m}_{n,\gn} &= \frac{1}{\ga_n - \ga_\gn}\pa{\pa{\ga_n - x_0} \gd^{m}_{n - 1,\gn} - m \gd^{m - 1}_{n - 1,\gn}} \nonumber \\
+&= \frac{1}{\pa{\gn - n} h} \pa{-n h \gd^{m}_{n - 1,\gn} - m \gd^{m - 1}_{n - 1,\gn}} \label{eqndelta}
+\end{align}
+
+It is possible to extract the powers of $h$ from the coefficients by defining, for $\gn < n$:
+\[
+\gl^{m}_{n,\gn} \DefineAs \gd^{m}_{n,\gn}\of{\VectorSymbol{\ga}} \pa{-1}^m h^m
+\]
+
+Substituting into equation (\ref{eqndelta}) we obtain:
+\[
+\gl^{m}_{n,\gn} = \frac{1}{n - \gn} \pa{n \gl^{m}_{n - 1,\gn} - m \gl^{m - 1}_{n - 1,\gn}}
+\]
+and we see that $\gl^{m}_{n,\gn}$ is independent from $x_0$ and $h$ and therefore that $\gd^{m}_{n,\gn}$ is independent from $x_0$.
+
+Note that the definition of $\gl^{m}_{n,\gn}$ extends immediately to $n = \gn$, that is to $\gl^{m}_{n,n}$, using equation (3.10) of \cite{Fornberg1988}.
+
+Now knowing that:
+\[
+\gd^{m}_{n,\gn}\of{\VectorSymbol{\ga}} = \pa{-1}^m \frac{\gl^{m}_{n,\gn}}{h^m}
+\]
+it is straightforward to rewrite equation (\ref{fornbergbackwardpreliminary}) to obtain equation (\ref{fornbergbackward}), thus demonstrating the lemma.
+
+% https://tex.stackexchange.com/questions/79194/misplaced-qed-symbol-after-displaymath-inside-item-of-inline-list
+\vspace{-\belowdisplayskip}\[\]
+\end{proof}
+\end{lemma}
+
+\begin{corollary}
+Applying the lemma to the second derivative $f\dersecond\of{x}$ gives the following result when $m ≥ 2$:
+\begin{equation}
+f^{(m)}\of{x_0} = \frac{\pa{-1}^m}{h^{m - 2}} \sum{\gn = 0}[n - 2]{\gl^{m - 2}_{n - 2,\gn} f\dersecond\of{x_0 - \gn h}} + \BigO\of{h^{n - m + 1}}
+\label{eqnfornbergbackward2}
+\end{equation}
+\end{corollary}
+
+\subsection*{A formula for symmetric linear multistep integrators}
+In this section we derive a formula suitable for computing the velocity of a body knowing its positions and accelerations at preceding times.  This is the formula we use after integration using a symmetry linear multistep integrator.
+
+\begin{proposition}
+Given a sufficient regular function $f\of{x}$ there exists a family of rational numbers $\gh_{n, \gn}$, with $0 ≤ \gn ≤ n - 2$, such that: 
```

Comment:
There exists a family […], with […], such that, given a […] function $f<del>\of{x}</del>$

Reply:
You missed sufficient/sufficiently.